File: app/src/main/java/tourtle/ticketing2/utils/InternetConnection.java
package tourtle.ticketing2.utils;

import android.content.Context;
import android.net.ConnectivityManager;

/**
 * Created by ONYX on 6/6/2017.
 */
public class InternetConnection {
    private Context context;

    public InternetConnection(Context context) {
        this.context = context;
    }

    public static boolean checkConnection(Context context) {

            ConnectivityManager cm = (ConnectivityManager) context.getSystemService(Context.CONNECTIVITY_SERVICE);
            return cm.getActiveNetworkInfo() != null;
    }

}

File: app/src/main/java/tourtle/ticketing2/utils/InternetConnection.kt
package tourtle.ticketing2.utils

import android.content.Context
import android.net.ConnectivityManager

/**
 * Created by ONYX on 6/6/2017.
 */
class InternetConnection(private val context: Context?) {

    companion object {
        fun checkConnection(context: Context?): Boolean {
            val cm = context?.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
            return cm.activeNetworkInfo != null
        }
    }

}